livre/eq/Channel.cpp

```diff
@@ -118,8 +147,9 @@ struct Channel::Impl
 public:
     explicit Impl( Channel* channel )
           : _channel( channel )
-          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME )
+          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME, INVALID_FRAME )
           , _progress( "Loading bricks", 0 )
+          , _frameCounter( INVALID_FRAME )
```

Comment:
I mean that the frame count is already part of FrameInfo, why do you need a separate member variable for the same?